app/controllers/chapters_controller.rb

```diff
@@ -91,10 +91,10 @@ def new
   # GET /work/:work_id/chapters/1/edit
   def edit
     if params["remove"] == "me"
-      @chapter.pseuds = @chapter.pseuds - current_user.pseuds
+      @chapter.authors_to_remove = current_user.pseuds
```

Comment:
I put it in the model now because (a) I realized that would work now and (b) Elz mentioned observers have recently gone bye-bye in Rails, so having fewer things in the observer seemed like something future!us would appreciate.